examples/client.go

```diff
@@ -1,28 +1,29 @@
 package main
 
 import (
-	"github.com/graarh/golang-socketio"
-	"github.com/graarh/golang-socketio/transport"
 	"log"
 	"runtime"
 	"time"
+
+	"github.com/paxosglobal/golang-socketio"
+	"github.com/paxosglobal/golang-socketio/transport"
 )
 
-type Channel struct {
+type ChannelClient struct {
```

Comment:
this isolated file worked without this change, but I was having some annoying IDE's error because in the server.go file (same package) there were 2 structs with the same name so I renamed it